bintray/move.rst

```diff
@@ -0,0 +1,104 @@
+
+Conan.io is now on Bintray
+==========================
+
+
+Central repositories
+--------------------
+
+Bintray host two central repositories:
+
+  - `conan-transit`_
+
+    This repository is an exact copy of the conan.io repository at **June 11, 2017 08:00 CET**.
+    It is a read-only repository, you can download any packages that it hosts, but you are not able to
+    upload packages to it.
+
+  - `conan-center`_
+
+    This repository has moderated, curated and well-maintained packages, and is the place where you can share
+    your packages with the community. To share your package, you upload it to your own (or your organization’s)
+    repositories and submit a request to include it in `conan-center`_. Check :ref:`Working with conan-center<conan_center_flow>`
+
+
+Using the new Bintray repositories
```

Comment:
I have already added this section in "creating packages" -> "uploading packages"